Syllables_Stats.py

```diff
@@ -0,0 +1,78 @@
+import requests
+import json
+
+api_url = 'http://127.0.0.1:5000'
+
+class Syllables:
+    def __init__(self, language):
+        self.language = language
+        self.all_words_route = api_url + "/" + self.language
+
+    def get_all_syllables(self):
+        r = requests.get(self.all_words_route)
+        dico = json.loads(r.content)
+        result = dico['result']
+        all_syllables_ipa = {}
+        counter = 0
+        else_counter = 0
+        else_words = {}
+        for word in result:
+            word_syllables_ipa = word['syllables_ipa']
+            word_syllables = word['syllables']
+            if len(word_syllables) == len(word_syllables_ipa):
+                for i in range(0, len(word_syllables)-1):
+                    syllable_ipa = word_syllables_ipa[i]
+                    syllable = word_syllables[i]
+                    try:
+                        all_syllables_ipa[syllable_ipa]
```

Comment:
you don't need a try catch. First you could check `if syllable_ipa not in all_syllable_ipa` and then check if the syllable is in too.... but you can also, and i would recomment it, use the defaultdict from the collections module. It exactly does what you are looking for. 
```
ipas = defaultdict(lambda: defaultdict(int))
ips[syllable_ipa][syllable] += 1
```
(it s a default dict of default dict of int)